test_schemas/en/test_thank_you_census_communal_establishment.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Actually, I was mistaken, we do have unit tests for handlers. Therefore, in my opinion, we should write unit tests for the `ThankYou` handler instead.

Reply:
Not really, I think we have one for the block factory. I think we need to be consistent and something that should be brought up in tech session if we want to do it. This does have coverage, and if we do want tests on all handlers you can put my face on it

Reply:
We started adding them, i.e. the Submission handler you recently introduced, so I don't see why we wouldn't want unit test for newly introduced ones?. 

A lot of the existing one don't have them because they weren't very unit testable at the point it was introduced due to app dependencies. We should add a card for that, but this was around that we started adding it, so why wouldn't we want unit test for new ones?

Reply:
I wouldn't say we have one for the submission handler, it just checks the submission_language_code nothing else and thats only because I couldn't get hold of it initially. I actually nearly deleted it when dumper was changed to use it then giving coverage 

The only other one, isn't actually testing any handlers, it only tests the factory and only for exceptions (a bit weird). With my pragmatic hat on I don't think removing tests and schemas that match the pattern for everything else and give us coverage, to write new ones is great use of time after the work has been done. However I would support this approach going forward if that decision is made before a story is started. Will get Mark to confirm this is a sensible approach.

Reply:
We don't usually specify in the card to "write unit test", it a given. I fold.